Source/Core/Common/SysConf.cpp

```diff
@@ -177,8 +187,10 @@ bool SysConf::Save() const
       const u8 data_size = static_cast<u8>(item.bytes.size());
       AppendToBuffer<u8>(&entries, data_size);
       entries.insert(entries.end(), item.bytes.cbegin(), item.bytes.cbegin() + data_size);
-      // Unused byte.
-      entries.insert(entries.end(), '\0');
+      if (item.name == "IPL.NIK")
```

Comment:
`0x80` is likely Parental Controls enabled status, as I have a `0x00` there after clearing the Parental Controls. That byte might contain more bitwise flags, though, with further purposes.